templates/configuration/graphdb-logback-configmap.yaml

```diff
@@ -1,6 +1,8 @@
 # Default configuration map for provisioning GraphDB repository.
 # To change it, prepare another configuration map and update "graphdb.repositoryConfigmap"
-{{- if .Values.deployment.logbackConfigMap }}
+{{- $configs := (.Values.graphdb.configs | default dict) }}
+{{- if $configs.logbackConfigMap }}
+{{- if eq $configs.logbackConfigMap "graphdb-logback-configmap" }}
```

Comment:
you can use 'and'. Not necessary but I have to mention it

Reply:
If $configs.logbackConfigMap doesn't exist and I change it to "and" instead of 2 different if's when it tries to evaluate the single if I'll get an error that it cannot compare a nil to string

Reply:
damn ... I though it would evaluate them 1 by 1 and stop if the first is not ok ...